deploy/dashboard.yaml

```diff
@@ -216,8 +216,8 @@ spec:
           subPath: config.yaml
           readOnly: true
       serviceAccountName: polaris-dashboard
-      nodeSelector: {}
-      tolerations: []
+      nodeSelector:
+      tolerations:
```

Comment:
are these changes intentional - it's not mentioned in the commit message and I wanted to assure it wasn't an IDE formatting thing so I can make sure mine is setup the same way

Reply:
I think this is due to a Helm update (I'm on `2.13.0` now)